package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hbs",
   "description": "Express.js template engine plugin for Handlebars",
-  "version": "4.0.6",
+  "version": "4.1.0",
```

Comment:
You don't actually bump the version in the PR; the release process `npm version minor` will do the bump and tagging.